Detections/ASimDNS/imDns_IPEntity_DnsEvents.yaml

```diff
@@ -0,0 +1,67 @@
+id: 67775878-7f8b-4380-ac54-115e1e828901
+name: (Preview) TI map IP entity to DnsEvents (Normalized DNS)
+description: |
+  'Identifies a match in DNS events from any IP IOC from TI
+  To use this analytics rule, make sure you have deployed the [ASIM normalization parsers](https://aka.ms/AzSentinelDns)'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: ThreatIntelligence
+    dataTypes:
+      - ThreatIntelligenceIndicator
+  - connectorId: ThreatIntelligenceTaxii
+    dataTypes:
+      - ThreatIntelligenceIndicator
+  - connectorId: DNS
+    dataTypes:
+      - DnsEvents
+queryFrequency: 1h
+queryPeriod: 14d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Impact
+tags:
+  - Id: 69b7723c-2889-469f-8b55-a2d355ed9c87
+    version: 1.0.0
+  - Schema: ASIMDns
+    SchemaVersion: 0.1.1
+query: |
+  let dt_lookBack = 1h;
+  let ioc_lookBack = 14d;
+  let IP_TI = (ThreatIntelligenceIndicator
+  | where TimeGenerated >= ago(ioc_lookBack) and ExpirationDateTime > now()
+  | where Active == true
+  // Picking up only IOC's that contain the entities we want
+  | where isnotempty(NetworkIP) or isnotempty(EmailSourceIpAddress) or isnotempty(NetworkDestinationIP) or isnotempty(NetworkSourceIP)
+  // As there is potentially more than 1 indicator type for matching IP, taking NetworkIP first, then others if that is empty.
+  // Taking the first non-empty value based on potential IOC match availability
+  | extend TI_ipEntity = coalesce(NetworkIP, NetworkDestinationIP, NetworkSourceIP,EmailSourceIpAddress,"")
+  | summarize LatestIndicatorTime = arg_max(TimeGenerated, *) by IndicatorId, TI_ipEntity);
+  let TI_IP_List=IP_TI | summarize make_set( TI_ipEntity);
+  imDns(starttime=ago(dt_lookBack), response_has_any=todynamic(toscalar(TI_IP_List)))
```

Comment:
We changed to response_has_any_prefix? didn't we? Also, using things, I feel we need to change to response_has_any_ipv4_prefix and response_has_ipv4 respectively.